src/helpers/calculateTimeLeft.ts

```diff
@@ -9,11 +9,11 @@ const calculateTimeLeft = (date: string): TimeLeft => {
   const difference =
     Number(new Date(date).getTime()) - Number(new Date().getTime());
 
-  // +18 hours is so that the countdown ends at 6pm on the launchDate shown
+  // +18 hours +60 minutes and +60 seconds so that the countdown ends at 6pm on the launchDate shown
```

Comment:
ive fixed these changes, the important thing is to add the 18 hours to the `const difference` above (where all our other values are derived from)

the below I dont think would work